app/controllers/step_by_step_pages_controller.rb

```diff
@@ -122,4 +128,9 @@ def set_step_by_step_page
   def step_by_step_page_params
     params.require(:step_by_step_page).permit(:title, :slug, :introduction, :description)
   end
+
+  def steps_in_step_by_step_page
+    @step_by_step_page = StepByStepPage.find(params[:step_by_step_page_id])
```

Comment:
`@step_by_step_page` is already being set in `set_step_by_step_page`, so you don't need to do it again here.